cordova-lib/package.json

```diff
@@ -19,6 +19,7 @@
   "dependencies": {
     "bplist-parser": "0.0.6",
     "cordova-js": "3.8.0",
+    "cordova-registry-mapper": "0.0.2",
```

Comment:
Yup. I could donate it to Apache if it makes sense. Right now it is easy for me to do releases for it without having to go through apache process. I imagine we will see some PRs coming in for it from other plugin devs who rename.